test/Microsoft.AspNet.Mvc.FunctionalTests/ModelBindingFromHeaderTest.cs

```diff
@@ -325,11 +325,12 @@ public class ModelBindingFromHeaderTest
             Assert.Equal("Title", error);
         }
 
-        // This model uses default value for 'Title'.
+        // This model uses [DefaultValueAttribute(...)] for 'Title', and the model binder won't 
+        // trounce it.
         //
-        // There's no validation error because we validate the default value.
+        // There's a validation error because we validate the value in the request (not present).
         [Fact]
-        public async Task FromHeader_BindHeader_ToModel_NoValues_DefaultValue_NoValidationError()
+        public async Task FromHeader_BindHeader_ToModel_NoValues_DefaultValueAttribute_ValidationError()
```

Comment:
I added that to be more explicit. When I've said 'default value' in the past, it's been mistaken for a value set in the constructor of the model.